lib/matplotlib/backends/backend_cairo.py

```diff
@@ -509,7 +476,7 @@ def print_svg(self, fobj, *args, **kwargs):
     def print_svgz(self, fobj, *args, **kwargs):
         return self._save(fobj, 'svgz', *args, **kwargs)
 
-    def _save (self, fo, format, **kwargs):
+    def _save(self, fo, fmt, **kwargs):
```

Comment:
I will grumble a bit about this, but it is OK.  It is a private API and the change is to avoid shadowing a builtin.